docs/blog/2020-03-11-netlify-functions-and-gatsby-cloud/index.md

```diff
@@ -107,15 +107,15 @@ In my opinion, one of the coolest things about Gatsby.js is that you can "hook i
 
 `onPostBuild` runs right after the build completes. We can use it to prepare and copy the functions over to the right place, before it's handed off to Netlify.
 
-Learn more about Gatsby build hooks in [our documentation](https://www.gatsbyjs.org/docs/node-apis/).
+Learn more about Gatsby build hooks in [our documentation](/docs/node-apis/).
 
 ### Zip It and Ship It
 
 Remember when I mentioned that Netlify brushes away the thorns of working with AWS Lambda? One of those thorns is the quirk that every function needs to be its own packaged project.
 
 Let's say we have two functions, `track-hit.js` and `like-content.js`. And let's assume that they both use `faunadb`, a Node module. We need to produce two `.zip` files, with the following contents:
 
-```
+```text
```

Comment:
```suggestion
```
```
Since this is showing a file structure we don't typically use a language tag.

Reply:
there are on all beginning code blocks in our docs are a language tag und for directory tree there is a `text`

i would like to have here a `directory`, `files` or a `tree` or something what is applicable

Reply:
Ya, those others are not available in markdown. You're correct that we've used text elsewhere so let's stick with that for now.

Reply:
FYI: `treeview` is on the way https://github.com/PrismJS/prism/issues/2264